github/enterprise_rules.go

```diff
@@ -76,14 +76,14 @@ func (s *EnterpriseService) UpdateEnterpriseRuleset(ctx context.Context, enterpr
 	return rs, resp, nil
 }
 
-// UpdateEnterpriseRulesetClearBypassActor clears the ruleset bypass actors for a ruleset for the specified repository.
+// UpdateRepositoryRulesetClearBypassActor clears the bypass actors for a repository ruleset for the specified enterprise.
 //
 // This function is necessary as the UpdateEnterpriseRuleset function does not marshal ByPassActor if passed as an empty array.
```

Comment:
Perhaps we should change the comment after renaming:
```suggestion
// This function is necessary as the UpdateRepositoryRuleset function does not marshal ByPassActor if passed as an empty array.
```

Reply:
Good spot, fixed.